### DISCUSSION TITLE
No "public" folder?

### DISCUSSION THREAD

**Discussion by taylorren** [symfony/symfony#45365] [Q&A]
I followed the instruction in Symfony official site to create a new project, using:

`symfony new my_project_directory --webapp`

After this command, my directory does NOT contain a `public` folder. Any hint on this?

![Ashampoo_Snap_2022年2月9日_12h34m40s_002_](https://user-images.githubusercontent.com/3853711/153122602-1958ead5-9adc-4e35-a8db-c01341b12d1f.png)

The folder structure is as attached.

**Comment by ahundiak** [comment]
Try creating a project with `symfony new --full project` and see what happens.

I suspect the --webapp option is not quite ready for the masses and might be failing on windows.  In particular, it tries to install the webpack bundle which can cause all kinds of interesting problems if the system is not properly setup.
